### ISSUE TITLE
evaluate.py support other data format?

### ISSUE THREAD

**Issue by peacherwu** [fizyr/keras-retinanet#911]
Is there a way I can run `evaluate.py` on a separate test set with trained models? It appears it doesn't support oid and a couple of other formats where `train.py` does.

**Comment by hgaiser** [comment]
Those datasets, like OID, have been added later via community contributions. If you want to make a PR to add them to `evaluate.py` then it would be much appreciated.

**Comment by beibeiZ** [comment]
how can I use evaluate.py? @

> Those datasets, like OID, have been added later via community contributions. If you want to make a PR to add them to `evaluate.py` then it would be much appreciated.



> Is there a way I can run `evaluate.py` on a separate test set with trained models? It appears it doesn't support oid and a couple of other formats where `train.py` does.
